App/Sagas/index.ts

```diff
@@ -81,6 +81,11 @@ import {
 
 export default function * root () {
   yield all([
+    call(manageNode),
+    call(onNodeCreated),
+    call(onNodeStarted),
+    call(onNodeOnline),
```

Comment:
Wire up the node lifecycle listeners

Reply:
Becoming a fan of not using `takeEvery` etc in the root saga. It's cleaner to just yield a `call()` and let the called function contain the `take` logic is wants. More self contained that way, and the root saga is more like a manifest with no logic.